index.html

```diff
@@ -112,27 +112,31 @@ <h5 class="cardtitle fw-bolder mx-2 text-center opacity-100">Network</h5>
       </div>
 
       <div class="card register-btn h-50 p-2 mx-auto d-flex flex-row justify-content-evenly align-items-center
-                rounded-0 mb-3">
-        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light text-decoration-none
+                rounded-0  my-4 ">
+        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light
           " href="#">Join Our Art Exhibition</a>
       </div>
```

Comment:
- [ ] Great job so far :muscle: Kindly ensure that `Join Our Art Exhibition` is underlined as shown in the Figma design as the previous reviewer mentioned. Please include that change as well :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/225444114-b07a9537-9481-440d-9911-05446adfc173.png)

- The Behance Design
![image](https://user-images.githubusercontent.com/102699958/225444664-5493c2e7-5e6b-44b1-aa0f-14650e53e4e5.png)

Reply:
@NahnahAJ, thanks for the review. I implemented all the changes as asked by the previous reviewer but I did not redeploy my project. I have a question, should I deploy the project using the branch I created the PR or from the main branch?

Reply:
Hi @rosemutai,  for deployment we usually use the main branch so that approved changes are deployed from the new branch when it is merged into it.